packages/graphback-core/tests/snapshots/RelationshipMetadataBuilderTest.ts.md

```diff
@@ -79,7 +77,9 @@ Generated by [AVA](https://ava.li).
          "start": 89,␊
          "end": 204␊
         }␊
-       }␊
+       },␊
+       "name": "sentMessages",␊
```

Comment:
Is this just a result of AST order being changed in GraphQL-JS?

Reply:
Yes. result.of downgrade